[PLATFORM-995]: Use OTel recommended names for attributes (#92)
primait/teleplug: CHANGELOG.md

## [Unreleased]

### Added

- Usage of the `opentelemetry_semantic_conversion` auto-generated library to ensure proper conventions are followed

### Changed

- Changed client attributes to proper server attributes
  - `"net.peer.ip"` is now `"net.sock.peer.addr"`
  - `"net.peer.port"` is now `"net.sock.peer.port"`

### Removed

> - `"http.host"` attribute removed from from `http_common_attributes` since it doesn't exist and is covered by `"net.host.name"`

Reviewer: Think this might be more understandable for the end user

```suggestion
- `"http.host"` attribute as it is non standard, and already covered by `"net.host.name"`
```
